includes/classes/traits/NotifierManager.php

```diff
@@ -126,16 +147,26 @@ protected function logNotifier($eventID, $param1, $param2, $param3, $param4, $pa
         error_log($zcDate->output("%Y-%m-%d %H:%M:%S") . ' [main_page=' . $main_page . '] ' . $eventID . $output . "\n", 3, $file);
     }
 
-    private function eventIdHasAlias($eventId)
+    private function eventIdHasAlias($eventId): bool
     {
         if (array_key_exists($eventId, $this->observerAliases)) {
             return true;
         }
+
```

Comment:
Why not just `return array_key_exists($eventId, $this->observerAliases);` directly?

Reply:
Ya, I was thinking the same. It was an existing method so I didn't change it. But I agree. Will update.

Reply:
Updated.